HandsOnLab/README.md

```diff
@@ -85,11 +85,11 @@ private void OnPropertyChanged([CallerMemberName] string name = null) =>
 private void OnPropertyChanged([CallerMemberName] string name = null)
 {
     var changed = PropertyChanged;
+    
+    if (changed == null)
```

Comment:
Sure, but this is an explicit C# 5 Sample for Visual Studio 2012 and 2013. The C# 6 sample right above this section uses the `?.` operator.